src/magnify/apps/core/views/room_token_view.py

```diff
@@ -53,8 +54,11 @@ class RoomTokenView(APIView):
     """View for token fetching based on a guest user and a specific room
     This redirects to jitsi with the token"""
 
+    permission_classes = (IsAuthenticated,)
```

Comment:
This is why your tests are failing. You should not force authentication, you should test in the view that if the user is connected and deduct from this what s.he can do or not.